chunky/src/java/se/llbit/chunky/ui/SceneChooserController.java

```diff
@@ -173,7 +178,11 @@ public void setStage(Stage stage) {
   private void populateSceneTable(File sceneDir) {
     List<SceneListItem> scenes = new ArrayList<>();
     List<File> fileList = SceneHelper.getAvailableSceneFiles(sceneDir);
-    Collections.sort(fileList);
+    // TODO: Sort by recently modified instead of file name.
+    Collections.sort(fileList, Comparator
+            .comparing((File o) -> -(Long)o.lastModified()) // negative of last modified -> most recent at top
```

Comment:
```suggestion
            .comparingLong(File::lastModified)
            .reversed() // most recent by file last modified at top
```

Reply:
We want to sort the latest modified file to the top of the list, I believe this flips the order. If you use method reference instead of method call, you can't negate the values.

Reply:
Reversed applies a flip on the previous comparator (which is lastModified), therefore the last recent files are still sorted at the top :)